CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Contributing to finance
+This is a learning space for django/react stack, and get used to be working with docker tools and a complete professional workflow with CI revisions, so everyone feel free to:
+
+-   Report a bug
+-   Discussing the current state of the code
+-   Submitting a fix
+-   Proposing new features
+-   Comment a better approach to any part of the code.
+
+## We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+## Workflow
+Pull requests are the best way to propose changes to the codebase.
+
+1.  Fork the repo if you're not a contributor.
+2.  Create your branch from `master`.
+3.  If you've added code that should be tested, add tests.
+4.  If you've changed APIs, update the documentation.
+5.  Ensure the test suite passes.
+6.  Make sure your code lints.
+7.  Issue that pull request!
+
+### PR format
+There is a PR template in the repo, and for the title please use:
+```
+[<issue number that the PR solves>] - Description
+```
+Make sure to use the keywords in the Issue field, like closes and related, also, please remember to use the correct labels.
+
+The PR will do a couple of checks, including CI bulid on TravisCI, Flake8 for python code, automated CR with codacy and coverage test with Codecov.
+
+## Any contributions you make will be under the GPL-3.0 License.
```

Comment:
Codacy found an issue: [[no-heading-punctuation] Don’t add a trailing `.` to headings](https://app.codacy.com/manual/lgruelas/finance/pullRequest?prid=5748300)